flink-contrib/flink-storm/src/test/java/org/apache/flink/storm/wrappers/StormTupleTest.java

```diff
@@ -595,7 +593,7 @@ public void testGetBinaryByFieldPojoGetter() throws Exception {
 	private <T> StormTuple testGetByField(int arity, int index, T value)
 			throws Exception {
 
-		assert (index < arity);
+		Assert.assertTrue(index < arity);
 
```

Comment:
Because they are disabled during run time. This makes bugs e.g. in stack traces harder to find.